src/content/en/2024/security.md

```diff
@@ -2,17 +2,1740 @@
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Host</th>
+        <th>Desktop</th>
+        <th>Mobile</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>www.gstatic.com</code></td>
+        <td class="numeric">35.27%</td>
+        <td class="numeric">35.43%</td>
+      </tr>
+      <tr>
+        <td><code>cdnjs.cloudflare.com</code></td>
+        <td class="numeric">7.31%</td>
+        <td class="numeric">7.35%</td>
+      </tr>
+      <tr>
+        <td><code>cdn.userway.org</code></td>
+        <td class="numeric">6.20%</td>
+        <td class="numeric">5.69%</td>
+      </tr>
+      <tr>
+        <td><code>static.cloudflareinsights.com</code></td>
+        <td class="numeric">6.07%</td>
+        <td class="numeric">5.97%</td>
+      </tr>
+      <tr>
+        <td><code>code.jquery.com</code></td>
+        <td class="numeric">5.36%</td>
+        <td class="numeric">5.58%</td>
+      </tr>
+      <tr>
+        <td><code>cdn.jsdelivr.net</code></td>
+        <td class="numeric">3.92%</td>
+        <td class="numeric">4.27%</td>
+      </tr>
+      <tr>
+        <td><code>d3e54v103j8qbb.cloudfront.net</code></td>
+        <td class="numeric">2.15%</td>
+        <td class="numeric">1.88%</td>
+      </tr>
+      <tr>
+        <td><code>t1.daumcdn.net</code></td>
+        <td class="numeric">1.82%</td>
+        <td class="numeric">1.14%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Most common hosts from which SRI-protected scripts are included", sheets_gid="1612151810", sql_file="TODO.sql") }}</figcaption>
+</figure>
+
+Most of the hosts from which resources are fetched and protected by SRI are CDNs. A notable difference from 2022's data is the absence of `cdn.shopify.com` from the top hosts list (previously 22% on desktop and 23% on mobile). This is due to Shopify having dropped SRI in favor of similar functionality provided by the `integrity` attribute of `importmap`, which they explain in a <a hreflang="en" href="https://shopify.engineering/shipping-support-for-module-script-integrity-in-chrome-safari#">blogpost</a>.
+
+### Permissions Policy
+
+The [Permissions Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Permissions_Policy) (formerly known as the Feature Policy) is a set of mechanisms that allow websites to control which browser features can be accessed on a webpage, such as geolocation, webcam, microphone, and more. By using the Permissions Policy, websites can restrict feature access for both the main site and any embedded content, enhancing security and protecting user privacy. This is configured through the `Permissions-Policy` response header for the main site and all its embedded `<iframe>` elements,. Additionally, web administrators can set individual policies for specific `<iframe>` elements using their `allow` attribute.
```

Comment:
Nit but be drop the locale for MDN links so they redirect to the best language if they include translations:

```suggestion
The [Permissions Policy](https://developer.mozilla.org/docs/Web/HTTP/Permissions_Policy) (formerly known as the Feature Policy) is a set of mechanisms that allow websites to control which browser features can be accessed on a webpage, such as geolocation, webcam, microphone, and more. By using the Permissions Policy, websites can restrict feature access for both the main site and any embedded content, enhancing security and protecting user privacy. This is configured through the `Permissions-Policy` response header for the main site and all its embedded `<iframe>` elements,. Additionally, web administrators can set individual policies for specific `<iframe>` elements using their `allow` attribute.
```